### PR TITLE
[Android] Remove BT from android codebase

### PR DESCRIPTION
**Pull request by juancruzgs**
#### Description:

Remove `bt` folders from the codebase since it's now in a separate repository
Moved things from the `gps` flavor to `main` since it's no longer needed to have a flavor

### CODE REVIEW

**Comment by juancruzgs** [comment]
@JacobJaffe Could you do a quick smoke test and verify it's working as expected?
I tested it and seems to be working ok, but I don't know every feature the app has

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,28 +1,43 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (juancruzgs) [inline android/app/src/main/AndroidManifest.xml:1]: I merged the AndroidManifest files that were in `src/main` and `src/gps`

File: android/app/build.gradle [hunk 2]
```diff
@@ -206,18 +196,6 @@ android {
...
-    productFlavors {
-        gps {
-            applicationId 'org.pathcheck.covidsafepaths'
-            dimension "version"
-        }
-        bt {
-            applicationId 'org.pathcheck.covidsafepathsBt'
-            dimension "version"
-        }
-    }
```
Comment (juancruzgs) [inline android/app/build.gradle:219]: We no longer need product flavors.
The new build variants are `debug`, `staging`, `release`

**Review by JacobJaffe — APPROVED** [review]
Awesome, thanks for this cleanup.
Ran smoke test through functionality, LGTM!

**Pull request merged**